### PR TITLE
Fix for GRD-68428

### PR DESCRIPTION
**Pull request by apurva-birajdar**
Fix for GRD-68428
[AWS OUA UC]: New S-TAP is getting created when traffic resumes on each DB reboot

Bug is verified by QA, publishing it to public repo.

### CODE REVIEW

**Review by RefaelAdi — APPROVED** [review]

**Review by ofer-haim — APPROVED** [review]

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+* Supported Guardium versions: Guardium Data Protection: 11.4 and above
 
 ## Requirements
 
 1. Unified auditing must be enabled in an Oracle database that will be monitored by this method
-2. Download the Basic Instant client package from Oracle. 
-Note: In this release only specific Instant client will be supported from v21.1.0.0.0 Download [here](https://download.oracle.com/otn_software/linux/instantclient/211000/oracle-instantclient-basic-21.1.0.0.0-1.x86_64.rpm)
-3. Download the OUA UC plugin `guardium-oua-uc.zip` from [here](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-oua-guardium/OracleUnifiedAuditPackage/OracleUnifiedAudit/guardium-oua-uc.zip)
+2. Download the Basic Instant client package from Oracle.
+   **Note:** : In this release only specific Instant client will be supported from v21.1.0.0.0 Download [here](https://download.oracle.com/otn_software/linux/instantclient/211000/oracle-instantclient-basic-21.1.0.0.0-1.x86_64.rpm)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:14]: In this release, 

instance clients?

starting v21.1.0.0.0.

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
+3. Download the OUA universal connector plug-in  `guardium-oua-uc.zip` from [here](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-oua-guardium/OracleUnifiedAuditPackage/OracleUnifiedAudit/guardium-oua-uc.zip)
 4. A designated user for OUA UC should be created for Oracle database access. An existing user with sysdba privileges can also be used
-5. A secret containing the user’s password for OUA UC must be created
-    - Example: `grdapi universal_connector_keystore_add key=OUA_USER_PASS password=<PASSWORD>` where `<PASSWORD>` is the OUA UC user’s password for the database. `OUA_USER_PASS` will be used in the plugin configuration as a variable for password secret
+5. A secret containing the user’s password for OUA universal connector must be created
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:17]: You must create a secret containing your OUA universal connector password.

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+    - Example: `grdapi universal_connector_keystore_add key=OUA_USER_PASS password=<PASSWORD>` where `<PASSWORD>` is the OUA universal connector user’s password for the database. `OUA_USER_PASS` will be used in the plug-in configuration as a variable for password secret
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:18]: is your password

for the password secret

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 4]
```diff
@@ -52,29 +53,43 @@ Update the variables in Makefile for your environment's Java home and Logstash l
 
 4. Click on the "UPLOAD” button and upload downloaded `oracle-instantclient-basic-21.1.0.0.0-1.x86_64.rpm` and then `guardium-oua-uc.zip`
 
-5. Click on "+". The Connector Configuration dialog opens 
+   5. Click on "+". The Connector Configuration dialog opens
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:56]: Click the plus icon.

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 5]
```diff
@@ -52,29 +53,43 @@ Update the variables in Makefile for your environment's Java home and Logstash l
 
 4. Click on the "UPLOAD” button and upload downloaded `oracle-instantclient-basic-21.1.0.0.0-1.x86_64.rpm` and then `guardium-oua-uc.zip`
 
-5. Click on "+". The Connector Configuration dialog opens 
+   5. Click on "+". The Connector Configuration dialog opens 
 
-    - Type any unique name in the "Connector name" field
+       - Type any unique name in the "Connector name" field
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:58]: **Connector name**

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 6]
```diff
@@ -52,29 +53,43 @@ Update the variables in Makefile for your environment's Java home and Logstash l
...
 
-    - Paste content of "oua_config" in the "Input configuration" field
+         - Paste content of "[ouaPipe.conf](https://github.com/IBM/universal-connectors/blob/main/filter-plugin/logstash-filter-oua-guardium/ouaPipe.conf)" in the **Input configuration** field
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:60]: Paste the content from the [] () file in the

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 7]
```diff
@@ -52,29 +53,43 @@ Update the variables in Makefile for your environment's Java home and Logstash l
...
+             ```
+           pipe {
+             type => "oua"
+             command => "${OUA_BINARY_PATH} -c ${THIRD_PARTY_PATH} -s ${THIRD_PARTY_PATH} -r 1 -t 1000 -p 10 -j <USER>/${OUA_USER_PASS}@<SERVER_ADDRESS>:<SERVER_PORT>/<INSTANCE_NAME>"
+                         add_field => {"SERVER_ADDRESS" => "<Enter_Server_Address>"}
+                             add_field => {"SERVER_PORT" => "<Enter_Server_Port>"} }
+           ```
 
-    - Paste content of "oua_filter" in the "Filter configuration" field
+       - Paste content of "[ouaPipe.conf](https://github.com/IBM/universal-connectors/blob/main/filter-plugin/logstash-filter-oua-guardium/ouaPipe.conf)" in the **Filter configuration** field
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:70]: Paste the content from the [] ( )file in the

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 8]
```diff
@@ -52,29 +53,43 @@ Update the variables in Makefile for your environment's Java home and Logstash l
...
+                       }
+           if "_jsonparsefailure" not in [tags] {
+               oua_filter {}
+           }
+         }
+
+           ```
 
-    - NOTE: The type specified for the filters must be unique among Universal Connectors and be identical in the input and filter configurations
+       - **NOTE**: The type specified for the filters must be unique among Universal Connectors and be identical in the input and filter configurations
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:88]: universal connectors,

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 9]
```diff
@@ -52,29 +53,43 @@ Update the variables in Makefile for your environment's Java home and Logstash l
...
 
-6. Click Save. Guardium validates the new connector and enables plugin. After it is validated, it appears in the Configure Universal Connector page.
+   6. Click **Save**. Guardium validates the new connector and enables plugin. After it is validated, it appears in the **Configure Universal Connector** page.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:90]: the plug-in. 

appears on the

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 10]
```diff
@@ -52,29 +53,43 @@ Update the variables in Makefile for your environment's Java home and Logstash l
...
 
-Note: The following arguments are used in configuration for OUA plugin
+**Note**: The following arguments are used in configuration for the OUA plug-in
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:92]: in the OUA plug-in configuration.

File: filter-plugin/logstash-filter-oua-guardium/README.md [hunk 11]
```diff
@@ -85,4 +100,4 @@ Note: The following arguments are used in configuration for OUA plugin
 
 ## Limitation
 
-Normally, the "Statement Type" attribute for the "FULL SQL" entity in Guardium reports shows us whether a full SQL statement is a prepared statement. However, because OUA doesn't give us information about whether a statement is a prepared statement or not, the "Statement type" attribute is not applicable for the OUA UC plug-in.
+Normally, the "Statement Type" attribute for the "FULL SQL" entity in Guardium reports shows us whether a full SQL statement is a prepared statement. However, because OUA doesn't give us information about whether a statement is a prepared statement or not, the "Statement type" attribute is not applicable for the OUA universal connector plug-in.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-oua-guardium/README.md:103]: `statement type`

**Pull request merged**